build/helpers/coverage.js

```diff
@@ -387,3 +464,90 @@ function isBlockBorder(input) {
 function isWhitespace(input) {
   return input === " " || input === "\t" || input === "\n" || input === "\r";
 }
+
+/**
+ * @param {Script} script
+ */
+function printCoverageStatistics(script) {
+  let file = "";
+  if (script.sources.length === 1) {
+    file += " in " + script.sources[0].path;
+  } else if (script.sources.length === 2) {
+    file += " in " + script.sources[1].path;
+  }
+  const total = calculateTotalCoverage(script);
+  if (total < 90) {
+    notify.warn(`Low coverage (${total.toFixed(2)}%)` + file);
+  }
+}
+
+/**
+ * @param {Script} script
+ * @param {FunctionCoverage | BlockCoverage} coverage
+ */
+function printCoverage(script, coverage) {
+  // Skip all blocks that are covered at least once.
+  if (coverage.count !== 0) {
+    return;
+  }
+
+  const { start, end } = coverage.range;
+
+  const source = script.sources.find(source => source.path === start.path);
+
+  if (source === undefined) {
+    return;
+  }
+
+  const uncovered = source.content.substring(start.offset, end.offset);
+
+  const filePath = path.relative(
+    process.cwd(),
+    path.resolve(script.base, source.path)
+  );
+
+  const above = `${source.lines[start.line - 1].value}`;
+  const below = `${source.lines[end.line + 1].value}`;
+
+  const before = source.lines[start.line].value.slice(0, start.column);
+  const after = source.lines[end.line].value.slice(end.column);
+
+  let output = `${chalk.bold("Suggested Block to Cover")}`;
+
+  output += `\n${chalk.dim(`${filePath}:${start.line + 1}-${end.line + 1}`)}`;
```

Comment:
You can leave out the end line number and just display the start.

Reply:
I actually think it makes better sense to show the end-line. We don't want them to cover line A, we want them to cover line A to B.

Reply:
The line number is just an indication of where to start looking for the code snippet, similar to how linting and compile errors are shown.